CHANGELOG.md

```diff
@@ -6,34 +6,33 @@
 
 ### Added
 
-- Added `RenderMask` as a replacement to camera filters to `Scene`, `Layer` and `Entity`. 
-- Added `Clear()` method to `ResourceBox`.
-- Added `Offset()` method to linear dampers.
-- Added `OnFrameStart`, `OnFrameFinish` and `OnAfterDraw` events to `GraphicsMgr`.
-- Added `Pool` collection.
-- Added `AccumulationBuffer` collection.
-- Added `UnorderedList` collection.
-- Added `GetArea()`, `GetSignedArea()`, `IsClockwise()` methods to `GameMath`.
-- Added new collision system.
-- Added new drawing methods.
+- Abstractions interfaces to handle platform-specific stuff and implemented abstractions for Windows (`WindowsClipboard`, `WindowsLocalStorage`, `WindowsMonitorFrequencyManager`, `WindowsScalingManager`, `WindowsTextInputBinder`, `WindowsWindowWatch` classes).
+- Added `PositionComponent` class and `Entity.GetPostition()` extension method.
+- Added `ConfigStash` class.
+- Added `BufferedInput` class.
+- Added `BeizerCurve` and `VerticalAxisProjection` methods to `GameMath` class.
+- Added `Sector` class.
+- Added `HotReloadWatch` class.
+- Added `LockableFlag` class.
 
 ### Changed
 
-- **BREAKING CHANGE:** `ResourceInfoMgr` now accepts wildcards instead of directory names. For example, `ResourceInfoMgr.GetResourcePaths("Graphics/Trees");` should now be replaced with `ResourceInfoMgr.GetResourcePaths("Graphics/Trees/*");`
-- **BREAKING CHANGE:** Renamed `GetSafeNormalize()` to `SafeNormalize()`.
-- **BREAKING CHANGE:** Removed instances of `Width` and `Height` in `Sprite`, `Frame`, `WindowMgr`, `Camera`, `Surface`, and replaced them with `Size`.
-- **BREAKING CHANGE:** Changed boolean `isOutline` to `ShapeFill` enum for shapes.
+- Migrated to NET8.
+- Updated MonoGame to `3.8.2`.
+- Updated Nopipeline to `2.3.0`.
+- **BREAKING CHANGE:** Locked OpenGL on Windows.
```

Comment:
"Removed Linux and Mac support form DesktopGL target. You will have to implement abstractions interfaces to make it work again."